internal/gui/assets/templates/index.html

```diff
@@ -62,8 +62,10 @@ <h2>Identify the Miner</h2>
                             </p>
                             <div class="d-flex flex-row">
                                 <ul>
-                                    <li>
-                                        Coming Soon!
+                                    <li><a
+                                        rel="noreferrer"
```

Comment:
We typically use `rel="noopener noreferrer"` on external links. `noopener` prevents the page in the newly opened tab from being able to access/manipulate the originating tab, e.g. it could redirect to a phishing page.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener

Reply:
I can add it if you want, but target="_blank" implicitly adds it per the docs.

Reply:
Added it even though, as mentioned, it was already implicit because of the target.

Reply:
Nice, I wasn't aware. Still nice to have just for consistency with other links/projects.